packages/dds/tree/src/shared-tree/sharedTree.ts

```diff
@@ -332,6 +370,35 @@ export class SharedTree
 		});
 	}
 
+	public exportVerbose(): VerboseTree | undefined {
```

Comment:
Docs for these?

Reply:
These are inherited members from an interface which provides docs. I had a bug where I forgot to declare that interface implementation which broke IntelliSense, but I have now fixed that.

Adding explicit comments here like inherit doc would be a down grade in my opinion since these are non exported APIs and breaking IntelliSense for them by using a syntax our API generator likes seems bad.